index.md

```diff
@@ -2,18 +2,26 @@
 layout: home
 ---
 
-<span style="font-size: 1.1em; font-weight: 400;">
-UAVCAN is a lightweight protocol designed for reliable communication in aerospace and robotic
-applications via [CAN bus](https://en.wikipedia.org/wiki/CAN_bus).
-</span>
+**UAVCAN is an open lightweight protocol designed for reliable communication in aerospace and robotic
+applications over robust vehicular networks such as CAN bus.**
 
-<hr/>
+- Democratic network -- no bus master, no single point of failure.
+- Publish/subscribe and request/response (RPC) exchange semantics.
+- Efficient exchange of large data structures with automatic decomposition and reassembly.
+- Lightweight, deterministic, easy to implement, and easy to validate.
+- Suitable for deeply embedded, resource constrained, hard real-time systems.
+- Supports dual and triply modular redundant transports.
+- Supports high-precision network-wide time synchronization.
+- High-quality open source reference implementations are freely available under the MIT license.
 
-- Democratic network - no single point of failure.
-- Publish/subscribe or request/response exchange semantics.
-- Allows efficient exchange of large data structures.
-- Lightweight, easy to implement protocol.
-- Supports network-wide time synchronization with microsecond precision.
-- Doubly- or triply- redundant CAN bus.
-- Can be used in deeply embedded, resource constrained, hard real-time systems.
-- The specification and high quality reference implementations are open and free to use ([MIT License](http://opensource.org/licenses/MIT)).
+This website is dedicated to the **experimental version** of the protocol, otherwise known as **UAVCAN v0**.
+There is work underway to release the first stable long-term protocol specification and a set of
+robust reference implementations by **2019 Q2**, at which point UAVCAN v0 will reach the end of its life cycle.
```

Comment:
I'm not on board for v0 "reaching the end of its life-cycle" in Q2 of 2019. I think it has reached the end of active development already and, by the transitive property, so have the reference implementations but the standard will only be "superseded" by the new standard. "end of lifecycle" implies that existing deployments are obsolete which, as the owner of an existing deployment, is somewhat offensive.

After defining that v1 supersedes v0 we can discuss v0 "end of life-cycle" as being at least 2-years out (2021). We should also clarify that the short EOL timeframe is because of the experimental nature of UAVCAN v0 and that v1 will receive longer Long Term Support (to borrow the Ubuntu "LTS" acronym).

Finally, let's not give the false impression that the v0 specification has any _fatal_ defects. While many of the standard datatypes haven't provided the utility expected the core protocol and some of the standard datatypes have proven to be completely functional.

Reply:
Please review.